pages/axios.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id="app">
```

Comment:
Nuxtでは要素が一つなのでわざわざidを設定しなくていい
```suggestion
  <div>
```